src/main/java/frc/robot/subsystems/arm/ArmIOSim.java

```diff
@@ -58,8 +57,14 @@ public void setElbowP(double kP) {
         elbowController.setP(kP);
     }
 
+
+    @Override
+    public String getSubsystemName() {
+        return "arm";
+    }
+
     @Override
-    public void updateInputs() {
+    public void updateInputs(ArmInputs inputs) {
```

Comment:
This isn't actually necessary. You could pass the inputs through the io constructor and have it as a member in the io class (both real and sim).